gapic/templates/tests/unit/gapic/%name_%version/%sub/test_macros.j2

```diff
@@ -1761,12 +1761,8 @@ def test_{{ method_name }}_rest_no_http_options():
 {% set transport_name = get_transport_name(transport, is_async) %}
 def test_transport_kind_{{ transport_name }}():
     {% if transport_name == 'rest_asyncio' %}
-    if not HAS_GOOGLE_AUTH_AIO:
-        pytest.skip("google-auth >= 2.35.0 is required for async rest transport.")
-    elif not HAS_AIOHTTP_INSTALLED:
-        pytest.skip("aiohttp is required for async rest transport.")
-    elif not HAS_ASYNC_REST_SUPPORT_IN_CORE:
-        pytest.skip("google-api-core >= 2.20.0 is required for async rest transport.")
+    if not HAS_ASYNC_REST_EXTRA:
+        pytest.skip("the library must be installed with `async_rest` extra to use this feature.")
```

Comment:
Could you file an issue to follow up on making sure or documenting how to see these skip messages? Also, mention that in some cases we should use `mark.skipif`: https://docs.pytest.org/en/stable/reference/reference.html#pytest-mark-skipif-ref

Reply:
Filed https://github.com/googleapis/gapic-generator-python/issues/2203